package.json

```diff
@@ -7,31 +7,42 @@
     "eslint",
     "eslintconfig"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/simplabs/eslint-config-simplabs.git"
+  },
+  "license": "MIT",
+  "author": "simplabs GmbH <info@simplabs.com>",
   "homepage": "https://github.com/simplabs/eslint-config-simplabs#readme",
   "bugs": {
     "url": "https://github.com/simplabs/eslint-config-simplabs/issues"
   },
-  "license": "MIT",
-  "author": "simplabs GmbH <info@simplabs.com>",
   "main": "configs/ember.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/simplabs/eslint-config-simplabs.git"
-  },
   "scripts": {
     "test": "node .",
     "nodetest": "mocha node-tests --recursive"
   },
   "dependencies": {
+    "ember-cli-htmlbars": "^4.0.0",
     "ember-cli-version-checker": "^3.1.3",
     "execa": ">=1.0.0",
     "fs-extra": "^8.1.0",
     "supports-color": "^7.1.0"
   },
   "devDependencies": {
-    "ember-cli": "~2.17.0",
+    "@ember/optional-features": "^1.0.0",
+    "babel-eslint": "^10.0.3",
+    "ember-cli": "~3.13.1",
     "ember-cli-blueprint-test-helpers": "^0.18.0",
+    "ember-cli-template-lint": "^1.0.0-beta.3",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-qunit": "^4.5.1",
+    "ember-source-channel-url": "^2.0.1",
+    "ember-try": "^1.2.1",
+    "eslint-plugin-ember": "^7.1.0",
+    "eslint-plugin-node": "^10.0.0",
```

Comment:
do we really need all of these new dependencies?

Reply:
I ran `ember-cli-update` which added all the default dependencies but of course we don't need them in this case - removed 👍